endoreg_db/views/video/correction.py

```
@@ -0,0 +1,530 @@
+    def post(self, request, pk):
+        """Apply masking to video."""
+        video = get_object_or_404(VideoFile, pk=pk)
+        
+        # Extract parameters
+        mask_type = request.data.get('mask_type', 'device')
+        device_name = request.data.get('device_name')
+        roi = request.data.get('roi')
+        processing_method = request.data.get('processing_method', 'streaming')
+        
+        # Validate required parameters
+        if mask_type == 'device' and not device_name:
```

Ruff (0.14.1)

Unpacked variable `metadata` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)